features/home/src/main/kotlin/com/teacherworkout/features/home/LandingScreen.kt

```diff
@@ -1,39 +1,111 @@
 package com.teacherworkout.features.home
 
-import androidx.compose.foundation.background
 import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.*
 import androidx.compose.material.MaterialTheme
 import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.res.colorResource
 import androidx.compose.ui.res.dimensionResource
 import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.input.TextFieldValue
 import androidx.compose.ui.tooling.preview.Preview
-import com.teacherworkout.features.learn.R
+import com.teacherworkout.features.home.data.HomeData.dummyLessonThemes
+import com.teacherworkout.features.home.data.LessonTheme
+import kotlin.random.Random
+
+val lessonThemes = mutableStateOf(dummyLessonThemes)
+val searchValue = mutableStateOf(TextFieldValue())
+val listState = LazyListState()
+
+//simulates the filtering of themes
+fun applyFilter(text: String) {
+    if (text.isBlank()) {
+        lessonThemes.value = dummyLessonThemes
+    } else {
+        val n = Random.nextInt(1, dummyLessonThemes.size)
+        val someLessonThemes = mutableListOf<LessonTheme>()
+        repeat(n) {
+            someLessonThemes.add(dummyLessonThemes[Random.nextInt(0, dummyLessonThemes.size)])
+        }
+        lessonThemes.value = someLessonThemes
+    }
+}
 
 @Composable
 fun LandingScreen() {
+    val space32dp = dimensionResource(id = R.dimen.space_32dp)
     val space16dp = dimensionResource(id = R.dimen.space_16dp)
-    val backgroundColor = colorResource(id = R.color.landing_background)
-    Box(modifier = Modifier.fillMaxSize()) {
-        Column(
+    val space8dp = dimensionResource(id = R.dimen.space_8dp)
+    Column(modifier = Modifier.fillMaxSize()) {
+        LazyColumn(
```

Comment:
Prety cool this lazyColumn :+1:

Reply:
It slipped.. Initially I had the themes title and the search view outside the LazyColumn and I didn't notice that I was left with only one composable in this Column. I fixed it now.